packages/apps/autorest/package.json

```diff
@@ -55,7 +55,7 @@
     "static-link": "^0.3.0",
     "@autorest/core": "~3.0.0",
     "@azure-tools/async-io": "~3.0.0",
-    "@azure-tools/extension": "~3.0.0",
+    "@azure-tools/extension": "~3.1.0",
```

Comment:
I believe this should be a real dependnecy of AutoRest since it gets used as part of the runtime code.

Reply:
yeah, i am still having issues, I think bundledDependencies is acting weird with pnpm

Reply:
Oh great...  I hadn't considered that possibility